arm-trafficmanager/2017-05-01/swagger/trafficmanager.json

```diff
@@ -0,0 +1,972 @@
+        },
+        "endpointLocation": {
+          "type": "string",
+          "description": "Specifies the location of the external or nested endpoints when using the ‘Performance’ traffic routing method."
+        },
+        "endpointMonitorStatus": {
+          "type": "string",
+          "description": "Gets or sets the monitoring status of the endpoint."
+        },
+        "minChildEndpoints": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets or sets the minimum number of endpoints that must be available in the child profile in order for the parent profile to be considered available. Only applicable to endpoint of type 'NestedEndpoints'."
+        },
+        "geoMapping": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Gets or sets the list of countries/regions mapped to this endpoint when using the ‘Geographic’ traffic routing method. Please consult Traffic Manager Geographic documentation for a full list of accepted values."
+        }
+      },
+      "description": "Class representing a Traffic Manager endpoint properties."
+    },
+    "Endpoint": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Gets or sets the ID of the Traffic Manager endpoint."
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets or sets the name of the Traffic Manager endpoint."
+        },
+        "type": {
+          "type": "string",
+          "description": "Gets or sets the endpoint type of the Traffic Manager endpoint."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/EndpointProperties"
+        }
+      },
+      "description": "Class representing a Traffic Manager endpoint."
+    },
+    "CheckTrafficManagerRelativeDnsNameAvailabilityParameters": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Gets or sets the name of the resource."
+        },
+        "type": {
+          "type": "string",
+          "description": "Gets or sets the type of the resource."
+        }
+      },
+      "description": "Parameters supplied to check Traffic Manager name operation."
+    },
+    "DnsConfig": {
+      "properties": {
+        "relativeName": {
+          "type": "string",
+          "description": "Gets or sets the relative DNS name provided by this Traffic Manager profile.  This value is combined with the DNS domain name used by Azure Traffic Manager to form the fully-qualified domain name (FQDN) of the profile."
+        },
+        "fqdn": {
```

Comment:
Is this supposed to be `readOnly: true` as the description suggests `This is formed from the...`